app/src/main/res/layout/fragment_destination_library.xml

```diff
@@ -69,10 +72,11 @@
     android:id="@+id/go_to_downloads_button_no_files"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:layout_marginTop="8dp"
     android:text="@string/download_books"
     android:visibility="gone"
-    app:layout_constraintEnd_toEndOf="parent"
-    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintEnd_toEndOf="@+id/file_management_no_files"
```

Comment:
these lines don't need to change

Reply:
I have changed the constraint's so that we can add or move some view / set of view's to the layout easily.
Also we can easily add animations to the view if we want by using KeyFrame Animations.